### PR TITLE
Fixes the broken pipeline - regressor tag to GradientBoostingQuantileRegressor

### PR DESCRIPTION
**Pull request by RuneChristensen-NN**
See title.

### CODE REVIEW

File: ProcessOptimizer/learning/gbrt.py [hunk 1]
```diff
@@ -46,6 +46,11 @@ def __init__(self, quantiles=[0.16, 0.5, 0.84], base_estimator=None,
         self.base_estimator = base_estimator
         self.n_jobs = n_jobs
 
+    def __sklearn_tags__(self):
```
Comment (SRFU-NN) [inline ProcessOptimizer/learning/gbrt.py:49]: Why isn't this automatic from RegressorMixin? https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/base.py line 614 should make this fix not needed.

Comment (SRFU-NN) [reply ProcessOptimizer/learning/gbrt.py:49]: Uh, I might have figured it out. From the docstring: "Mixin classes should always be on the left-hand side for a correct MRO". But BaseEstimator is on the left of RegressorMixin in the class inheritance.

So it should work if we just switch BaseEstimator and RegressorMixin in line 15

Comment (RuneChristensen-NN) [reply ProcessOptimizer/learning/gbrt.py:49]: We do something funky at lines  107-109 in learning/cook_estimator.py for the specific estimator, that caused the trouble.

image

Maybe we lose some inheritance somewhere along the way

Comment (SRFU-NN) [reply ProcessOptimizer/learning/gbrt.py:49]: Switching the inheritance order works, see  #302

**Comment by RuneChristensen-NN** [comment]
Closing request as #302 is a better solution solving the root of the error

**Pull request closed without merging**